.travis.yml

```diff
@@ -1,8 +1,9 @@
+dist: trusty
 language: scala
 scala:
-- 2.11.11
+- 2.12.10
```

Comment:
Do we need this at all for a java project?

Reply:
This project use sbt to build the java project, so I think Travis needs scala.